c2corg_ui/static/partials/feedtable.html

```diff
@@ -0,0 +1,66 @@
+<div ng-show="wfeedCtrl.documents.length > 0"
+  infinite-scroll="wfeedCtrl.getDocumentsFromWhatsnewFeed()"
+  infinite-scroll-immediate-check="false"
+  infinite-scroll-distance="0.7"
+  infinite-scroll-disabled="wfeedCtrl.busy || wfeedCtrl.error || wfeedCtrl.feedEnd || wfeedCtrl.noFeed">
+
+  <div class="table-responsive">
+    <table class="text-center feedtable table table-condensed table-striped table-hover">
+
+      <p class="text-success text-center" ng-if="wfeedCtrl.userId"
+         ng-init="user = wfeedCtrl.userId;" ng-model="amount = wfeedCtrl.feedLength;">
+        This feed for the user {{user}} contains {{amount}} changes.
+      </p>
+      <p class="text-success text-center" ng-if="!wfeedCtrl.userId"
+         ng-model="amount = wfeedCtrl.feedLength;">
+        This feed contains {{amount}} changes.
+      </p>
+
+      <tr>
+        <th class="text-center" translate>title</th>
+        <th class="text-center" translate>type</th>
+        <th class="text-center" translate>written_at</th>
+        <th class="text-center" translate>quality</th>
+        <th class="text-center" translate>username</th>
+        <th class="text-center" translate>comment</th>
+      </tr>
+
+      <tr class="feed-list"
+              ng-repeat="doc in wfeedCtrl.documents track by doc.version_id"
+          ng-init="type = wfeedCtrl.documentType(doc.document.type); console.log(type);" >
+
+        <td class="text-left">
+          <a href="{{wfeedCtrl.getDocHref(doc)}}">{{::doc['document']['title']}}</a> ({{::doc['lang']}})
```

Comment:
I see a problem with image with no title => it's not possible to click on the title because there is... no title :P 
We need a default (i18n) string when no title is available. Perhaps "Untitled image"?